eigerco/axelar-amplifier
# Solana verifier

## Description
Functionality that adds Solana verifier functionality

## Todos


## Steps to Test
- Run tofnd locally
- Run ampd locally
- Start verification by using properly deployed and configured contracts
- Check if poll is started
- End the poll
- Check the poll results

## Expected Behaviour
Poll results should be correct, based on the msg sent for verification

## Other Notes
